dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/FileReport.java

```diff
@@ -0,0 +1,44 @@
+package edu.harvard.iq.dataverse.api;
+
+import edu.harvard.iq.dataverse.dataset.DatasetReportService;
+import edu.harvard.iq.dataverse.persistence.user.AuthenticatedUser;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.StreamingOutput;
+import java.time.LocalDateTime;
+
+@Path("reports/file")
```

Comment:
I find this naming convention quite misleading.. When first read i thought we'll be downloading some reports for a single file (!?). Also: why limit whole 'reports' endpoint to single report type (namely: file report only)?

Hence I would rather go for something like "report" (as a verb) here and on the `createFileReport` I would change path to "/files".

Reply:
Good point – but I think it's better to leave "reports" in plural, so if in future we add a new type of report we won't have inconsistent naming in api.